### PR TITLE
Create sender.py

### PR DESCRIPTION
**Pull request by StevenTeglman**

### CODE REVIEW

File: sender.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import socket
+import sys 
+import json
+import subprocess
+# IP address and port of the receiver
+ip_address = '130.225.39.77'  # Replace this with the recipient's IP address
+port = 80  # Replace this with the recipient's port number
+
+pi_private_key = ''
```
Comment (StevenTeglman) [inline sender.py:9]: We should standardize using " instead of '

File: sender.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+pi_public_key = ''
+
+with open('/etc/wireguard/public-key-matrix-synapse', 'r') as file:
+    # Read the first line
+    pi_public_key = file.readline()
+
+with open('/etc/wireguard/private-key-matrix-synapse', 'r') as file:
+    # Read the first line
+    pi_private_key = file.readline()
+
```
Comment (StevenTeglman) [inline sender.py:19]: why two newlines? Please format properly

File: sender.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    client_socket.connect((ip_address, port))
+
+    # Send the message
+    client_socket.sendall(message.encode())
+
+    # Receive data from the server (optional)
+    response = client_socket.recv(1024)
+    print('Received:', response.decode())
+    received_msg = json.loads(response.decode())
+
```
Comment (StevenTeglman) [inline sender.py:44]: 3 new lines? Come on

File: sender.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    # Line number you want to replace
+    line_number = 2
+
+    # New text to replace the line with
+    new_text = 'Address = ' + response.decode() + '/24'
+
+    # Read the content of the file into memory
+
+
```
Comment (StevenTeglman) [inline sender.py:62]: what the fuck is this. Why so many newlines?

File: sender.py [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+                '\n',
+                '[Peer]\n',
+                'PublicKey = ' + received_msg['proxy_public_key'] + '\n',
+                'Endpoint = 130.225.39.77:51820\n',
+                'AllowedIPs = 10.0.0.0/24, 192.168.1.0/24\n',
+                'PersistentKeepalive = 25']
+        file.writelines(lines_to_add)
+
+
+
```
Comment (StevenTeglman) [inline sender.py:78]: Are you autistic?

**Review by StevenTeglman — COMMENTED** [review]
please address changes

**Review by LasseRSoerensen — APPROVED** [review]
this is amazballs

**Pull request merged**